verse/Dockerfile

```diff
@@ -27,37 +27,24 @@ RUN apt-get update \
     ssh \
     ## for building pdfs via pandoc/LaTeX (These are large!)
     lmodern \
```

Comment:
Need to verify if `lmodern` is really needed (I'm not sure), because TinyTeX has included the `lm` package.

Reply:
yeah, good call.  The fact that I still have packages pulling in texlive should raise suspicions, that's one of them to be sure (depends on `tex-common`).  Lemme do a quick test w/o it.